modules/data.land/R/cohort2pool.R

```diff
@@ -30,27 +30,63 @@ cohort2pool <- function(veg_file, allom_param = NULL) {
   
   ## Grab allometry
   if(is.null(allom_param)){
-    a <- 2                        
-    b <- 0.3
+    a <- -2                        
+    b <- 2.5
   } else {
     print("user provided allometry parameters not yet supported")
     return(NULL)
   }
   
-  #Calculate AGB
+  #Calculate Total Biomass
+  #Need to Convert the total kg of biomass to (kg/m^2) but data on area of land does not exist
+  #In it's place,dividing by number of trees.
   biomass = 10^(a + b*log10(dbh))
   biomass[is.na(biomass)] <- 0
-  tot_biomass <- sum(biomass)
-  AGB <- tot_biomass
+  tot_biomass <- sum(biomass)/length(biomass) # units (KgC/m^2)
+
+  
+  #Calculate Component Biomass
+  #Jenkins, Jennifer C., et al. "Comprehensive database of diameter-based biomass regressions for North American tree species." 
+  # Gen. Tech. Rep. NE-319. Newtown Square, PA: US Department of Agriculture, Forest Service, Northeastern Research Station.(2004).
+  #ratio = exp(B0 + (B1/DBH))
+  
+  component_names <- c("foliage","coarse_root", "stem_bark","stem_wood")
+  
+  # Hardwood params
+  hwood_0params <-c(-4.08113,-1.6911,-2.0129,-0.3065)
+  hwood_1params <- c(5.8816, 0.8160, -1.6805,-5.4240)
+  hlist <-list(hwood_0params, hwood_1params)
+  # Softwood params
+  swood_0params <-c(-2.9584,-1.5619,-2.0980,-0.3737)
+  swood_1params <- c(4.4766,0.6614,-1.1432,-1.8055)
+  slist <-list(swood_0params, swood_1params)
+  
+  
+  comp_ratios <- purrr::pmap(hlist,function(B0,B1){
+    
+    exp(B0 + (B1/dbh))
```

Comment:
why are you dividing B1/dbh. Power Law allometries take the form exp(B0 + B1*dbh). Again, we should probably just switch to using the allom module's predict function.